Eatery Blue/UI/EateryScreen/MenuCategoryPickerView.swift

```diff
@@ -7,6 +7,12 @@
 
 import UIKit
 
+protocol MenuCategoryPickerDelegate: AnyObject {
+
+    func menuCategoryPicker(buttonPressedAtIndex idx: Int)
```

Comment:
Maybe change the term idx to just "i" or "index" for the parameter name. I thought this had something to do with an Id, so it'll be cleaner to keep things consistent. I believe a lot of swift functions just use "index"

Reply:
Thanks for the feedbacck :))))